README.md

```diff
@@ -1464,15 +1428,102 @@ stock awards")).
   So, if you want to immediately exercise an option and file a Section 83(b) election, and
   you might have liquidity soon, it’s better if you can have it be an NSO.
 
+🚧 Although we refer to “restricted stock awards” in this section and once in 83b, so
+clarify that choice as well.
+We refer to “restricted stock” many times but its defining occurrence does not include a
+secondary referent of “restricted stock award.”
+
+### The AMT trap
```

Comment:
We have some more specifics (re: time of year you exercise ISOs in relation to triggering AMT and when it's too late to do anything about it) in Dummies notes. I'd like to flesh out this section a bit more! The why and wherefore of AMTs is also interesting, and the history of it helped me understand some of the details. J I'd suggest also a tax expert on this but I can add some more coverage in for the moment.

Reply:
Sure. An AMT-only PR just dealing with AMT, with a little history (with a link?) in the tax section and any focused details only would be helpful (separate from any other language/style PR) and then we can get review.